Update documentation for manage downtime page (#11495)
DataDog/documentation: content/en/monitors/downtimes.md

{{< img src="monitors/downtimes/downtime_examplebyname1_monitor.jpg" alt="downtime example"  style="width:80%;">}}

> 4. To schedule a downtime on more than one group (e.g. `service:synthesizer`, `service:consul`, etc.), you can create an additional downtime per group.

Reviewer: ```suggestion
4. To schedule a downtime on more than one group (such as `service:synthesizer` and `service:consul`), you can create an additional downtime per group.
```

## Manage

The Manage Downtime page displays the list of active and scheduled downtimes. Select a downtime to view details, edit, or delete it. Details include its creator, its scope, and a list of the monitors it applies to.
Use the facets panel and the search bar to filter the list on the `Creator`, the `Scope`, `Monitor Tags`, or `Active`, `Automuted`, `Recurring` parameters.

> {{< img src="monitors/downtimes/downtime_manage.png" alt="manage downtime page"  style="width:100%;">}}

Reviewer: ```suggestion
{{< img src="monitors/downtimes/downtime_manage.png" alt="manage downtime page" style="width:100%;">}}
```
